priv/rel/files/boot

```diff
@@ -187,10 +189,10 @@ generate_sys_config() {
                 __conform="$ROOTDIR/bin/conform"
             fi
 
-            # remove old sys.config since conform seems to merge this one too
-            rm -v "$GENERATED_CONFIG_DIR/sys.config"
+            # remove old sys.config first from the mutable dir since conform also considers this file for merging
+            rm -v $SYS_CONFIG
```

Comment:
we should instead remove $GENERATED_CONFIG_DIR (but only when derived from $RELEASE_MUTABLE_DIR)